app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -38,7 +38,15 @@
         "create-review-step-title": "Review Actions",
         "create-security-step-title": "Specify Security",
         "create-details-step-title": "Review/Edit Connector Details",
-        "edit-tooltip": "{{project.name}} cannot make this specification available for review/edit. If you want to review or edit the API before {{project.name}} creates the connector, try manually downloading the specification and then upload that file."
+        "edit-tooltip": "{{project.name}} cannot make this specification available for review/edit. If you want to review or edit the API before {{project.name}} creates the connector, try manually downloading the specification and then upload that file.",
+        "api-file-upload": "Upload an OpenAPI file",
+        "api-file-upload-dnd": "Drag and drop your OpenAPI file here or click the 'Choose File' button to use the file chooser",
```

Comment:
There should be a period at the end of that message since it is a complete sentence.

Reply:
@TovaCohen just checking to see if you saw this comment? Sorry if I missed your reply :)

Reply:
Thanks Michael, I did miss your comment. I think that the text here should match the text in the integration and extension importers.